modules/backend/controllers/Auth.php

```diff
@@ -52,6 +52,10 @@ public function index()
      */
     public function signin()
     {
+        if (BackendAuth::user()) {
+            return Backend::redirect('backend');
```

Comment:
The user doesn't necessarily have access to the dashboard, so it might be nice at some point to provide a helper method that would return the first menu item they have access to and use that instead, but this is probably good enough for now given that the current behaviour is just showing them the signin page when they're already logged in.